egs2/librispeech/asr2/run.sh

```diff
@@ -28,7 +28,7 @@ src_case="rm"
 tgt_case="ts"
 
 ./asr2.sh \
-    --kmeans_opts "--batch_bins 4800000" \
+    --kmeans_opts "--batch_bins 4800000 --nj 4" \
```

Comment:
You may need to increase `--num_threads` to deal with large memory consumptions in scikit-learn?
(Ideally, I want you to solve it by avoiding using such less-refined implementation.

Reply:
OK. Working on this item.